plugins/zenoh-rest/Cargo.toml

```diff
@@ -35,7 +35,7 @@ crate-type = ["cdylib"]
 zenoh = { version = "0.5.0-beta.5", path = "../../zenoh" }
 zenoh-router = { version = "0.5.0-beta.5", path = "../../zenoh-router" }
 zenoh-protocol = { version = "0.5.0-beta.5", path = "../../zenoh-protocol" }
-async-std = { version = "=1.8.0", features = ["unstable"] }
+async-std = { version = "=1.9.0", features = ["unstable"] }
```

Comment:
features = ["unstable"] was needed until async-std 1.8.0 to support the async channels. 
Since the channels have been stabilised in async-std 1.9.0, features = ["unstable"] should be hence removed.